easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.20-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,110 @@
+easyblock = 'Bundle'
+
+name = 'FusionCatcher'
+version = '1.20'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ndaniel/fusioncatcher'
+description = """FusionCatcher searches for novel/known somatic fusion genes, translocations,
+and chimeras in RNA-seq data (paired-end or single-end reads from Illumina NGS
+platforms like Solexa/HiSeq/NextSeq/MiSeq/MiniSeq) from diseased samples."""
+github_account = 'ndaniel'
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Java', '11', '', True),
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAT', '3.5'),
+    ('Bowtie', '1.2.3'),
+    ('Bowtie2', '2.3.5.1'),
+    ('BWA', '0.7.17'),
+    ('Kent_tools', '20190326', '-linux.x86_64', True),
+    ('SRA-Toolkit', '2.10.4'),
+    ('STAR', '2.7.2b'),
+    ('picard', '2.21.6', '-Java-11', True),  # Optional
+    ('Velvet', '1.2.10', '-mt-kmer_191'),  # Optional
+    ('OpenPyXL', '2.6.4'),  # Optional
+    ('parallel', '20190922'),  # Optional
+    ('pigz', '2.4'),
+    ('zlib', '1.2.11'),
+]
```

Comment:
From deps:
 ('BLAT', '3.5'), is included in ('Kent_tools', '20190326', '-linux.x86_64', True),
('BWA', '0.7.17'), is not used
('Velvet', '1.2.10', '-mt-kmer_191'), is not used
('parallel', '20190922'), is not used

Reply:
* your are right and several tools provided by BLAT are also available in Kent_tools, I'll check if we can reduce the number of dependencies
* BWA is used, please check for instance `fusioncatcher.py` around line 12239
```
if use_bwa:                            
    bd = outdir("focus_bwa.%d/" % (i,))
    job.add(_BA_+'bwa',kind='program')
```
* Velvel can also be used within FusionCatcher, please check for instance `fusioncatcher.py` around line 2593
```
if options.assembly:
    # save version of VELVET used to analyze the data
    job.add('printf',kind='program')                 
    job.add('"\nVELVET:\n------\n"',kind='parameter')
    job.add('>>',info_file,kind='output')
    job.run()
    job.add(_VT_+'velvetg',kind='program')
```
* parallel can also be used, please check for instance `fusioncatcher.py` around line 2377
```
if not os.system("which parallel 2>&1 >/dev/null"):
    job.add(_PL_+'parallel',kind='program')
```

Reply:
There is discussion in https://github.com/ndaniel/fusioncatcher/issues/163 
You might add you observations there.

Reply:
But since dependencies are not installed by neither `boostrap.py` nor `install_tools.sh` indicates they can't be used even though there are some left-overs in code?

Reply:
These dependencies are listed in the manual (https://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md), are in the bootstrap script (https://github.com/ndaniel/fusioncatcher/blob/master/bin/bootstrap.py) and appear in the code as I showed. So I really have zero doubts about this.
They are indeed optional, and they are marked as such in this easyconfig.

Reply:
They are left in bootstrap script, but they are not downloaded, if you run it, for example here:
https://github.com/ndaniel/fusioncatcher/blob/master/bin/bootstrap.py#L1739-L1755
And the author of the FusionCatcher said that they are not used.

But it can't hurt to add them...